recipes/dcmtk/all/conanfile.py

```diff
@@ -21,10 +22,8 @@ class DCMTKConan(ConanFile):
         "charset_conversion": [None, "libiconv", "icu"],
         "with_libxml2": [True, False],
         "with_zlib": [True, False],
-        "with_openjpeg": [True, False, "deprecated"],
```

Comment:
is it breaking for consumer?

Reply:
Yes. If they are using this option, they will break. Keep deprecated to avoid problems.

https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#can-i-remove-an-option-from-recipe

Reply:
From the same doc: 
> After one month, we will welcome a PR removing the option that was deprecated.

At some point in time, we need to remove things that are marked as deprecated.

Reply:
Fair enough.